Add filter data change admonition (#4567)
nautobot/nautobot: nautobot/docs/release-notes/version-2.0.md

#### Corrected Filter Fields ([#2804](https://github.com/nautobot/nautobot/pull/2804))

!!! warning

>     This change may introduce breaking changes to your existing `DynamicGroup`, Object Permission, `Relationship` filters, and any other saved references to these fields. You should review these models before upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#corrected-filter-fields) guide for more details.

Reviewer: ```suggestion
    This change may introduce breaking changes to your existing `DynamicGroup` filters, `ObjectPermission` filters, `Relationship` filters, and any other saved references to these fields. You should review any existing instances of these models before and after upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#corrected-filter-fields) guide for more details.
```

#### Enhanced Filter Fields ([#2804](https://github.com/nautobot/nautobot/pull/2804))

!!! warning

>     This change may introduce breaking changes to your existing `DynamicGroup`, Object Permission, `Relationship` filters, and any other saved references to these fields. You should review these models before upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#enhanced-filter-fields) guide for more details.

Reviewer: ```suggestion
    This change may introduce breaking changes to your existing `DynamicGroup` filters, `ObjectPermission` filters, `Relationship` filters, and any other saved references to these fields. You should review any existing instances of these models before and after upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#enhanced-filter-fields) guide for more details.
```

#### Renamed Database Foreign Keys and Related Names ([#2520](https://github.com/nautobot/nautobot/issues/2520))

!!! warning

>     This change may introduce breaking changes to your existing `DynamicGroup`, Object Permission, `Relationship` filters, and any other saved references to these fields. You should review these models before upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#renamed-database-fields) guide for more details.

Reviewer: ```suggestion
    This change may introduce breaking changes to your existing `DynamicGroup` filters, `ObjectPermission` filters, `Relationship` filters, and any other saved references to these fields. You should review any existing instances of these models before and after upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#renamed-database-fields) guide for more details.
```

#### Renamed Filter Fields ([#2804](https://github.com/nautobot/nautobot/pull/2804))

!!! warning

>     This change may introduce breaking changes to your existing `DynamicGroup`, Object Permission, `Relationship` filters, and any other saved references to these fields. You should review these models before upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#renamed-filter-fields) guide for more details.

Reviewer: ```suggestion
    This change may introduce breaking changes to your existing `DynamicGroup` filters, `ObjectPermission` filters, `Relationship` filters, and any other saved references to these fields. You should review any existing instances of these models before and after upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#renamed-filter-fields) guide for more details.
```

#### Redundant Filter Fields ([#2804](https://github.com/nautobot/nautobot/pull/2804))

!!! warning

>     This change may introduce breaking changes to your existing `DynamicGroup`, Object Permission, `Relationship` filters, and any other saved references to these fields. You should review these models before upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#removed-redundant-filter-fields) guide for more details.

Reviewer: ```suggestion
    This change may introduce breaking changes to your existing `DynamicGroup` filters, `ObjectPermission` filters, `Relationship` filters, and any other saved references to these fields. You should review any existing instances of these models before and after upgrading your production environment for any potentially subtle change. Please refer to the [Upgrading from Nautobot v1.X](../user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md#removed-redundant-filter-fields) guide for more details.
```
